docs/sources/alerting/unified-alerting/alerting-rules/create-grafana-managed-rule.md

```diff
@@ -112,6 +112,36 @@ The following template variables are available when expanding annotations and la
+| toLower            | string                     | string                 | strings.ToLower, converts all characters to lower case.                                                                            |
+| match              | pattern, text              | boolean                | regexp.MatchString Tests for a unanchored regexp match.                                                                            |
+| reReplaceAll       | pattern, replacement, text | string                 | Regexp.ReplaceAllString Regexp substitution, unanchored.                                                                           |
+| graphLink          | expr                       | string                 | Not supported                                                                                                                      |
+| tableLink          | expr                       | string                 | Not supported                                                                                                                      |
+| args               | []interface{}              | map[string]interface{} | Converts a list of objects to a map with keys, for example, arg0, arg1. Use this function to pass multiple arguments to templates. |
+| externalURL        | nothing                    | string                 | Returns a string representing the external URL.                                                                                    |
+| pathPrefix         | nothing                    | string                 | Returns the path of the external URL.                                                                                              |
+| tmpl               | string, []interface{}      | nothing                | Not supported.                                                                                                                     |
+| safeHtml           | string                     | string                 | Not supported.                                                                                                                     |
+| query              | query string               | []sample               | Not supported.                                                                                                                     |
+| first              | []sample                   | sample                 | Not supported.                                                                                                                     |
+| label              | label, sample              | string                 | Not supported.                                                                                                                     |
+| strvalue           | []sample                   | string                 | Returns the "\_\_value\_\_" label.                                                                                                 |
```

Comment:
I think we can safely say this not implemented yet and add it to the list of things to implement in the issue?